x-pack/plugins/security_solution/public/common/components/landing_cards/index.tsx

```diff
@@ -108,11 +84,17 @@ export const LandingCards = memo(() => {
             />
           </EuiFlexItem>
           <EuiFlexItem>
-            <EuiToolTip content={tooltipContent} position="top">
-              <EuiLink href="https://www.elastic.co/security" external={false} target="_blank">
-                <StyledEuiImage alt={i18n.SIEM_HEADER} size="xl" margin="l" src={imgUrls.video} />
-              </EuiLink>
-            </EuiToolTip>
+            <iframe
```

Comment:
imo we should add more attributes to this for security purposes, I know some customers don't want to leak the url they run kibana on, consider adding https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-referrerpolicy referrerpolicy="no-referrer" and probably the maximum set of sandbox attributes that will enable this to still work, maybe just sandbox without specifying, which would apply all, or some subset of these https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox

Reply:
thanks! done, needed `sandbox="allow-scripts allow-same-origin"`